pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/content/I18nTemplateConstants.java

```diff
@@ -11,6 +11,7 @@ public static final class Snapshot {
         public static final String PARTICIPANT_LAST_NAME = "DDP_PARTICIPANT_LAST_NAME";
         public static final String PARTICIPANT_BIRTH_DATE = "DDP_PARTICIPANT_BIRTH_DATE";
         public static final String PARTICIPANT_TIME_ZONE = "DDP_PARTICIPANT_TIME_ZONE";
+        public static final String DDP_SUBMISSION_DATE = "DDP_SUBMISSION_DATE";
```

Comment:
Let's rename to DDP_DATE_FIRST_COMPLETED . More consistent with what this really is.

Reply:
Should this change be applied to all `submissionDate` references?